frontend/public/components/resource-quota.jsx

```diff
@@ -406,6 +500,13 @@ export const ResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(
         namespaced: false,
         optional: true,
       });
```

Comment:
You shouldn't need to request both `ClusterResourceQuota` and `AppliedClusterResourceQuota`. You should only request one or the other.

Reply:
@spadgett - So do you not want to show `ClusterResourceQuota` on this page? In an earlier comment, you said you always wanted to show `AppliedClusterResourceQuota.`

Reply:
If this page is namespaced, it is probably better to show `AppliedClusterResourceQuota` so that you can show usage within the project.

On the `AppliedClusterResourceQuota` details page, I would link to the `ClusterResourceQuota` for users who have permission since `ClusterResourceQuota` is the editable resource.